app/services/medication_reminders.rb

```diff
@@ -2,7 +2,7 @@
 
 class MedicationReminders
   def send_take_medication_reminder_emails
```

Comment:
We should also create a rake task for running send_take_medication_reminder_emails in schedule.rake so that we can run this in production.

Reply:
I believe already there is a rake task scheduled for this in scheduler.rake, 
https://github.com/ifmeorg/ifme/blob/master/lib/tasks/scheduler.rake#L21
I have modified the existing task to consider weekly reminders. 
Let me know if I am missing something.

Reply:
Oh you're correct! Whoops sorry about that haha. I was being extra cautious it seems haha. I would run that rake command locally just to make sure things still work!